internal/server/v1/firehose/mappings.go

```diff
@@ -66,9 +68,12 @@ func mapFirehoseEntropyResource(def models.Firehose, prj *shieldv1beta1.Project)
 	}, nil
 }
 
-func makeConfigStruct(cfg *models.FirehoseConfig) (*structpb.Value, error) {
+func makeConfigStruct(cfg *models.FirehoseConfig, createdAt time.Time) (*structpb.Value, error) {
 	var stopTime *time.Time
-	if cfg.StopTime != nil {
+	if strings.ToUpper(cfg.EnvVars[confSinkType]) == "LOG" {
+		t := createdAt.Add(logSinkTTL)
```

Comment:
this shall be time.Now().Add(logSinkTTL)

Reply:
@ishanarya0  createdAt is nothing just time.Now() only. Being passed as param here.

Reply:
Yeah, i mean in all the 3 cases, we are adding 24hrs to current time. So, I think we need not to pass this current time(createdAt) as a param.

Reply:
got you. But earlier for 1 case, (edit), we were not extending it by 24hr. We were maintaining the existing created time only. But now this is not the case, we can remove params.